### DISCUSSION TITLE
Allow omitting `boot` and `enable` pins for `Expander` module

### DISCUSSION THREAD

**Discussion by falkoschindler** [zauberzeug/lizard#54] [Ideas]
In case users doesn't need to `flash()` or `disconnect()` an expander, they should be able to omit the `boot` and `enable` parameters:

```py
expander = Expander(serial, [boot, enable])
```

**Comment by JensOgorek** [comment]
I will have a look at this.

**Comment by JensOgorek** [comment]
I created a pull request (#55) with the modified expander
